frontend/javascripts/types/schemas/dataset_view_configuration_defaults.ts

```diff
@@ -71,6 +98,7 @@ export const enforceValidatedDatasetViewConfiguration = (
           : _.pickBy(layerConfigDefault, (value: any) => value !== null);
       }
     });
+    ensureDatasetSettingsHasLayerOrder(datasetViewConfiguration, dataset);
```

Comment:
> Code looks good, but I couldn't change the layer order for this DS: https://updatedefaultlayerconfiginviewmode.webknossos.xyz/datasets/sample_organization/idr0101A-4d/edit
even though it has multiple color layers?

Thanks for finding this bug :mag:.

This line should fix the bug (according to my testing). 
The problem was I added the new property `colorLayerOrder` (see https://github.com/scalableminds/webknossos/pull/7188) and that it is expected to be always present in a validated `DatasetConfiguration`. But the backend does not send this property if it wasn't set by the frontend previously. 
Previous to my changes, the property was only enforced during model initialization (see changes in `model initialization`). But the enforcing of the property with correct values must be done every time when working with a `DatasetConfiguration`. Thus it made more sense to me, to move the enforcing code to the `enforceValidatedDatasetViewConfiguration` function. 
This covers the bug, as this function is explicitly called in `dataset_settings_view.tsx` after it is fetched from the backend. 
Additionally, the `enforceValidatedDatasetViewConfiguration` is also enforced for the model initialization as it is part of the `getDatasetViewConfiguration` API backend call in `admin_rest_api.ts` which is called during the model initialization to fetch the `DatasetConfig` from the backend.

Could you please retest the changes (once the CI is done)?

Reply:
Hm, it still doesn't work for me [here](https://updatedefaultlayerconfiginviewmode.webknossos.xyz/datasets/sample_organization/idr0101A-4d/edit)? :thinking: Does it work for you?

Reply:
Great, now it works :)